vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+---
+title: GitHub Integration
+description: Information on the available vcpkg integration points with GitHub
+author: michaelbprice
+ms.author: miprice
+ms.date: 06/15/2023
+---
+
+# Available GitHub Integrations
+
+## Caching vcpkg-built binaries for your GitHub Actions workflows
+
+Repositories that use GitHub Actions for continuous integration can take advantage of [vcpkg's binary caching feature](./users/binarycaching.md) to reduce the amount of time it takes to build their projects. There are two binary cache providers that are available to GitHub repositories.
+
+### The GitHub Actions cache provider
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+The [GitHub Actions cache](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows) is intended to store a repository's intermediate build files that don't change often between jobs or workflow runs. GitHub provides a few different [tools to manage your Actions caches](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#managing-caches), which include REST APIs and an extension to the gh command line tool, so that you can optimize caches for your workflow. vcpkg's integration with GitHub Actions cache is through the [`x-gha` binary source provider](./users/binarycaching.md#gha).
```

Comment:
Can you do the https link without the language specifier (en)? I don't know if your repo is machine translated to other languages, but the link forces the English version on the reader. (I don't know if the GitHub docs are localized, either, but we generally avoid embedding language links to maximize the odds that foreign language readers will get the best experience.)

Reply:
The checks that run on our private repo catch this kind of thing. I'm not sure why your repo wasn't set up this way.